docs/contributor/kcp-sync-gardener-seed.md

```diff
@@ -0,0 +1,129 @@
+# Introduction
+
+This document contains the results of an investigation for a possible solution to provide available \
+Gardener Seeds region information for Kyma Control Plane (KCP) services as requested in [issue #629](https://github.com/kyma-project/infrastructure-manager/issues/629).
+
+---
+# Current architecture
+
+The Kyma Runtime is created on top of the Gardener System as a Gardener Shoot cluster. \
+For every Kubernetes Shoot cluster, its Control Plane services are located in a separate Kubernetes cluster named Gardener Seed. \
+There can be multiple Gardener Shoots managed by the single Gardener Seed cluster.
+
+<img alt="shoot.png" height="300" src="./assets/shoot.png" width="600"/>
+
+The set of active Seed clusters is managed on the Gardener System level. \
+During shoot creation, the Gardener system assigns a Seed for a Shoot based on internal configuration and availability of resources. \
+When the Shoot is fully created, the information about used Seed can be read from the Shoot CR instance.
+
+> Note:
+> Kyma Control Plane provisioning services may actually specify the Seed name for Shoot during creation, but they provide initial empty value
+> as they are not aware which Gardener seeds are available in the Gardener System.
+
+---
+# Problem description:
+
+The number of available Seed clusters is limited, and they exist only in several cloud provider regions. \
+It is highly probable that for provisioned Kyma Runtime the actual Seed cluster (assigned by Gardener) will be located in a different cloud region than Shoot cluster. \
+Such a situation can be problematic. \
+In some usecases users require Kyma Runtime to be fully located in the same cloud provider region — both underlying Gardener Shoot and its Seed. \
+There is a special checkbox to enforce this requrement during the initial setup of the Kyma Runtime.
+
+![seed_same_region_check.png](./assets/seed_same_region_check.png)
+
+As of now, when this checkbox is enabled, the KEB service is not able to validate if actual Gardener Seed is available in the chosen region before provisioning starts. \
+In such a case - provisioning operation fails because of missing Seed, and this fact is reported to the user after two hours after when provisioning times out.
```

Comment:
```suggestion
In such a case, the provisioning operation fails because of a missing Seed, and this fact is reported to the user after two hours after when provisioning times out.
```